generic-client/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "solana-generic-client"
+publish = false
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+chrono = { workspace = true }
```

Comment:
A lot of these dependencies are not really directly required, please clean up and only retain the necessary ones